### DISCUSSION TITLE
Nvidia Graphics under ChromeOS/Brunch (Just starting the discussion)

### DISCUSSION THREAD

**Discussion by Warrentheo** [sebanc/brunch#871] [Ideas]
I know that Nvidia graphics are not supported, and about the general headaches associated with Nvidia drivers under Linux...  I have a Ryzen3xxx CPU and wanted to try and get the Zork image to work, however I also have an RTX2070 as the only graphics option on this laptop...  I am not really asking to add this as a feature, but since it is possible to load the proprietary Linux driver into the Kernel under Ubuntu/Mint, what would be involved in trying to get it to work in the Brunch Kernel?  I assume that the nouveau driver would be buggy, and not worth the effort...

I am new to builds and such, but don't mind rolling my own build and doing some experiments, just curious at this point...  Any pointers or guidance would be appreciated...

**Comment by sebanc** [comment]
Brunch kernel has all graphic drivers enabled. What's missing is userspace drivers, for that you would need to build from source at least minigbm, libdrm and mesa with nvidia support.

Brunch purpose is not to build chromeos from source so it's outside the scope of this project.

Note: forgot to mention that even if you would do that the android container would not work as it would also need nvidia support.

**Reply by Warrentheo** [reply]
Ah, sad but thanks for the reply...
Love Brunch BTW :-D

**Comment by kentblake** [comment]
I am going to give this method a whirl.  I have not yet read much yet into the kernel includes as well as base requirements and gaps. I have an AMD 2700x and Nvidia GeForce 1060.  I have booted enough Chromium OS to know that I will need to build the the Drivers from SRC, along with perhaps dkms?  I do have some experience to get by in this endeavor.  If I make any progress, I will report back here. with a repo and Mimosas.

**Comment by Abdalla200-dot** [comment]
hoi
